scripts/inapp/capacitor.sh

```
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -ex;
+
+export VERSION=$(grep '^version:' pubspec.yaml | sed -e 's/version: //')
+
+mkdir -p inapp-sdks;
+
+cd inapp-sdks;
+
+export WORK_DIR="$(pwd)";
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)